testsuite/integration/microprofile/pom.xml

```diff
@@ -391,6 +391,13 @@
             <artifactId>wildfly-naming-client</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.jboss.arquillian</groupId>
+            <artifactId>arquillian-testcontainers</artifactId>
+            <version>1.0.0.Alpha1</version>
```

Comment:
FWIW I think for now this is fine while we test it, but this should move to the test-bom at some point.

Reply:
Agreed, but standard-test/pom.xml just has the Arquillian BOM, so maybe this should be added there, at which point the version HERE could be removed.